internal/controller/watchlist.go

```diff
@@ -0,0 +1,73 @@
+package controller
+
+import (
+	"Tradeasy/internal/services/watchlist"
+	"github.com/gin-gonic/gin"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+func CreateWatchlist(c *gin.Context) {
+	var req watchlist.CreateRequest
+	err := c.BindJSON(&req)
+	if err != nil {
+		return
+	}
+	res, err := watchlist.CreateWatchlist(req)
+	if err != nil {
+		log.Println(err.Error())
+		c.AbortWithStatus(http.StatusNotFound)
+	} else {
+		c.JSON(http.StatusOK, res)
+	}
+}
+
+func AddStockEntry(c *gin.Context) {
+	var req watchlist.AddStockRequest
+	watchlistId := c.Params.ByName("watchlist_id")
+	err := c.BindJSON(&req)
+	if err != nil {
+		return
+	}
+	i, _ := strconv.ParseInt(watchlistId, 10, 64)
+	res, er := watchlist.AddStockEntry(req, int(i))
```

Comment:
Modify er name.

Reply:
Please check all files and make this change.

Reply:
done